www/assets/js/components/SearchPage.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState, useCallback } from "react"
 import InfiniteScroll from "react-infinite-scroller"
 import { useCourseSearch } from "@mitodl/course-search-utils"
+import { serializeSort } from "@mitodl/course-search-utils/dist/url_utils"
```

Comment:
I'm not sure if this is an issue with the `serializeSort` function in `course-search-utils` or something here, but I was testing the search terms indicated in the attached issue and I wasn't able to get the desired results.  From the issue:

"sort by "MIT course nr" sorts using the value of course_id, ascending. For example, the results for [this search](https://ocwnext.odl.mit.edu/search/?d=Physics&q=%22quantum%20mechanics%22) should start with 8.04, 8.05, 8.06 and 8.07."

When I search for "quantum mechanics," the following are my first results when sorting by MIT course nr:

![image](https://user-images.githubusercontent.com/12089658/122255458-1fe12180-ce9c-11eb-8e37-f4eab6bfcd45.png)

Furthermore upon scrolling down, it seems that an attempt is made to sort the results in an ascending manner, but the course number seems to be being treated as a string or something as the order I'm getting has "1" results first, then "10."  For example, 10.34 follows 1.978.  The order I'm seeing in these search results by department number goes 1, 10, 12, 8, 9, ES, HST, IDS, MAS, RES.  So, it seems to be running the sort but is interpreting 10 and 12 to be less than 8 or 9.

Reply:
I'm also noticing that the courses noted in the issue, 8.04, 8.05, 8.06 and 8.07, do not show up in my search results.  The only results in department 8 that I get are 8.901 and 8.962.  This issue may be a larger issue with search and unrelated to the sorting functionality, but I thought it would be worth bringing up.

I'm also now seeing large inconsistencies in the search results.  I tried from the start again, typing "quantum mechanics" into the search bar and then switching the sort.  This time, departments 8 and 9 are not in my results.  It goes 1, 10, 12, MAS, etc.  Leaving the search term in the box and the sort dropdown on "MIT course nr" and clicking Search again, I get different results.  As I'm scrolling down, I can see it loading in a bunch of results from department 8, but when I scroll down I don't see them loaded in.  The results list goes from 6 to MAS.  Again, this might be an issue with the search API in `open-discussions`, but either way the departments in the teens (10, 12) should come *after* department 9 and before the lettered departments like MAS.

Reply:
After talking with @pdpinch and having him clarify the desired results here, he noted that in his search he was filtering on the Physics department manually by checking the facet.  This is what I get upon doing that:

![image](https://user-images.githubusercontent.com/12089658/122264028-160fec00-cea5-11eb-921d-73d520f3eb19.png)

As you can see the results are much more relevant but there seems to be a course from department 5 leaking into the results.  This seems to be unrelated to these changes as it happens on production too, so I'll make a separate issue for that.  As far as the sort issue, Peter and I also chatted about that and you can feel free to punt on it for now if it's too complex to solve.  I think it would be an issue in `course-search-utils` anyway, because that's where the actual sorting is happening right?  Anyway, based on the clarifications that have been made I'm okay merging this as most of what I've seen are issues unrelated to the changes being made here.  However, I think it's worth looking into seeing if we can easily change the sorting functionality to sort numbers correctly.  If it's not easy, we should just log the issue and move on for now.

Reply:
I don't think there's an easy way to address the numerical sorting problem. We could write a [painless script](https://www.elastic.co/guide/en/elasticsearch/reference/5.5/search-request-sort.html#_script_based_sorting) to be passed to elasticsearch when sorting, but I think a better option is to add a separate field which manipulates the `coursenum` field to make it sortable. I think that's outside this issue though.

Do you think the other issues are related to this PR?

Reply:
Another thing I'm seeing here though, is that the number of search results that used to be where the sort dropdown is now is missing?  I see it on prod, but not when running locally from this branch:

prod:
![image](https://user-images.githubusercontent.com/12089658/122265151-65a2e780-cea6-11eb-9056-255657c3e780.png)

gs/sort:
![image](https://user-images.githubusercontent.com/12089658/122265087-52901780-cea6-11eb-9aad-8cce68d50385.png)

Reply:
Yes, it was not in the design in the issue so I removed it

Reply:
Okay, so to recap:

 - There seems to be an issue with the infinite scroll component that causes an inconsistent amount of search results to actually render on the page that's unrelated to the changes in this PR and I logged an issue for that here: https://github.com/mitodl/ocw-hugo-themes/issues/114
 - The numerical sort issue is definitely a thing that should be solved, but as @noisecapella noted it's outside the scope of this PR
 - The amount of search results returned is normally displayed at the top of the list, but this PR removes it because it wasn't in the design @noisecapella was working with.  Does this seem correct, @pdpinch?

Reply:
> The amount of search results returned is normally displayed at the top of the list, but this PR removes it because it wasn't in the design @noisecapella was working with. Does this seem correct, @pdpinch?

Sorry, this was an oversight. Is it hard to put that back in?

Reply:
It shouldn't be hard to add back in, but the question of where to put it remains.  In the design, the sort dropdown kind of sits on top of the place it was in.  We could try and squeeze it in to the left of the sort dropdown, but that's not doing to work at mobile resolutions.

Reply:
My vote is to add it back on the next line, directly above the search result list.  I think that might be the safest option for now considering the mobile layout.

Reply:
That makes sense, I'll add it back on the right side of its own line